.env.example

```diff
@@ -31,6 +31,7 @@ RELAYER_QUERIES_TASK_QUEUE_CAPACITY=10000
 RELAYER_CHECK_SUBMITTED_TX_STATUS_DELAY=10s
 RELAYER_INITIAL_TX_SEARCH_OFFSET=0
 RELAYER_PROMETHEUS_PORT=9999
+RELAYER_IGNORE_ERRORS_REGEX=(execute wasm contract failed)
```

Comment:
please don't forget to update readme and documentation

Reply:
sure, as soon as we merge, will update docs